huawen-lei/week7/wed/homework-w7d3/js/main.js

```diff
@@ -0,0 +1,143 @@
+console.log('flickr search')
+
+//https://www.flickr.com/services/rest/?method=flickr.photos.search&api_key=2f5ac274ecfac5a455f38745704ad084&text=ocean+coral&format=json&nojsoncallback=1
+
+const FLICKR_BASE_URL = "https://www.flickr.com/services/rest/";
+const FLICKR_API_KEY = "2f5ac274ecfac5a455f38745704ad084";
+const pageNum = 1;
+
+
+
+$(function(){
+
+  $('#query').focus();
+  
+  buttonGenerate(5);
```

Comment:
why generate this before hand?

Reply:
every time i search, it keeps generating buttons and adding them on

Reply:
you can comment out this line as you have it on line 20 - which is where i would put it